### ISSUE TITLE
Did not work?  Result folder contains .mp4 with audio only?

### ISSUE THREAD

**Issue by Thermalzombie** [HypoX64/DeepMosaics#131]
Spent ages running this program and the result folder only contains a .mp4 file with aac audio.

Could the result be somewhere else?

Please helpt took hours on cpu version.

**Comment by robertdu86** [comment]
no, as far as i know the final file is always in the result folder, and yes cpu version is very slow compare to the gpu version. i already used it with at least 20 videos and i never had any problems until now, it's a little sad this project is not updated anymore because it is super good.
